### ISSUE TITLE
Mura ORM synthesized methods

### ISSUE THREAD

**Issue by damien-git** [blueriver/MuraCMS#2612]
This is either a bug or an error in the documentation.

The [Mura ORM documentation for Mura 7](http://docs.getmura.com/v7/mura-developers/mura-beans-objects/custom-objects/mura-orm/mura-orm-entity/) states that some methods are automatically added to components.

But in practice, this line appearing twice in bean.cfc restricts the synthesized methods to some components:
`if(listFindNoCase('content,user,feed,category,address,site,comment',prop.cfc)){`

Either the code or the documentation should be fixed so that they are consistent.

**Comment by mattlevine** [comment]
Can you post the property definition and how you are accessing the synthed method?

**Comment by damien-git** [comment]
No, I don't have any code left doing that. I realized the `remove#entityName#` method was not useful to me anyway because the entity gets deleted, and I just wanted to delete links. But just looking at the code for `getProperties()` in `bean.cfc`, I see that `remove#entityName#` does not get synthesized for other entities than the ones listed above, and this is not explained in the documentation.
